models/source_list.py

```diff
@@ -731,6 +739,20 @@ def get_downstreams(self, session=None):
 
         return wcs + zps
 
+    def show(self, **kwargs):
```

Comment:
Same comment about model/view separation as for the image show method.

A utility worth having (though I probably wouldn't do it here) is converting source positions to a ds9 region file.  It's easy to do (so I guess make an image), though you do have to remember that ds9 region files assume 1-center-offset pixels, not 0-center-offset pixels.

Reply:
That seems like something that could live down at the end of our issues list.